README.md

```diff
@@ -1,3 +1,12 @@
+
+⚠️ ⚠️ ⚠️
+
+**Deprecation Notice**
+
+The _OneSignal-Alexa-Nodejs-SDK_ has been deprecated. OneSignal no longer provides official support for Alexa. The last version released for Alexa >= x was v0.9.0. Please do not use _OneSignal-Alexa-Nodejs-SDK_ in any future projects; use [OneSignal API](https://documentation.onesignal.com/reference/create-notification) instead.
```

Comment:
Note that I don't know what version of Alexa we supported or if there were versions. If there is no relevant version, feel free to remove the `>= x` bit.

Reply:
Lets remove the >= X bit

Reply:
Fixed in https://github.com/OneSignal/OneSignal-Alexa-Nodejs-SDK/pull/3/commits/62a67784c68acf463982afc715f229706b43950d.